### DISCUSSION TITLE
XXZ Heisenberg tests

### DISCUSSION THREAD

**Discussion by RichRick1** [theochem/ModelHamiltonian#63] [Q&A]
It looks like [here](https://studenttheses.uu.nl/bitstream/handle/20.500.12932/36243/thesis%20Kuijper.pdf?sequence=1&isAllowed=y) starting from chapter 4 authors provide numerical solutions for the XXZ Heisenberg model for different number of sites (Chapter 6). 

Are there any other systems where we can check XXZ model, except the BETHE ANSATZ of anti-ferromagnetic chain 
@PaulWAyers @gabrielasd

**Comment by PaulWAyers** [comment]
There is the Richardson model, which is also an XXZ model. Also the picket-fence/Lipkin model which is a special case of the Richardson-Gaudin model. Paul Johnson could give us some data on that or we could use his code to generate our own. 

Some of these models are sort of cases where we benefit by the fact we have several spin models which we can write as seniority-zero models and vice versa, so testing of their compatability provides its own test.
